test/e2e/test_auditlogs.go

```diff
@@ -33,8 +33,60 @@ import (
 	_ "k8s.io/client-go/plugin/pkg/client/auth/gcp"
 )
 
-// SmokeCloudAuditLogsSourceTestImpl tests if a CloudAuditLogsSource object can be created to ready state and delete a CloudAuditLogsSource resource and its underlying resources..
-func SmokeCloudAuditLogsSourceTestImpl(t *testing.T, authConfig lib.AuthConfig) {
+// SmokeCloudAuditLogsSourceTestHelper tests if a CloudAuditLogsSource object can be created to ready state.
+func SmokeCloudAuditLogsSourceTestHelper(t *testing.T, authConfig lib.AuthConfig, cloudAuditLogsSourceVersion string) {
+	t.Helper()
+	client := lib.Setup(t, true, authConfig.WorkloadIdentity)
+	defer lib.TearDown(client)
+
+	project := os.Getenv(lib.ProwProjectKey)
+
+	auditlogsName := helpers.AppendRandomString("auditlogs-e2e-test")
+	svcName := helpers.AppendRandomString(auditlogsName + "-event-display")
+	topicName := helpers.AppendRandomString(auditlogsName + "-topic")
+	resourceName := fmt.Sprintf("projects/%s/topics/%s", project, topicName)
+
+	if cloudAuditLogsSourceVersion == "v1alpha1" {
+		lib.MakeAuditLogsV1alpha1OrDie(client, lib.AuditLogsConfig{
```

Comment:
can you just create this lib.AuditLogsConfig once outside the if, and then use it in all the cases?